### PR TITLE
Feat/add chain exchange module support

### PR DESCRIPTION
**Pull request by aarmoa**
- Added support in the SDK for all chain exchange module queries.
- Added unit tests and example scripts for all chain exchange module queries.
- Added support in the SDK for all chain exchange module requests.
- Added unit tests and example scripts for all chain exchange module requests.
- Refactored the chain example scripts to move them into separated subfolders for each module.
- Refactored all chain exchange module messages creation functions in the Composer to follow the Pythonic naming convention. Marked the original functions as deprecated.

### CODE REVIEW

**Comment by aarmoa** [comment]
@coderabbit review

**Comment by aarmoa** [comment]
@coderabbit resume

**Comment by aarmoa** [comment]
@coderabbitai review

File: pyinjective/async_client.py [hunk 1]
```diff
@@ -639,6 +646,392 @@ async def fetch_delegator_withdraw_address(
...
+
+    async def fetch_chain_spot_orderbook(
+        self,
+        market_id: str,
+        order_side: Optional[str] = None,
+        limit_cumulative_notional: Optional[str] = None,
+        limit_cumulative_quantity: Optional[str] = None,
+        pagination: Optional[PaginationOption] = None,
+    ) -> Dict[str, Any]:
+        # Order side could be "Side_Unspecified", "Buy", "Sell"
```
Comment (keithbaum) [inline pyinjective/async_client.py:759]: Side_Unspecified???

**Review by keithbaum — APPROVED** [review]

**Review by keithbaum — APPROVED** [review]
EXTENSIVE test coverage :)

**Pull request merged**